src/bin/stratisd.rs

```diff
@@ -412,12 +405,6 @@ fn run(matches: &ArgMatches, buff_log: &buff_log::Handle<env_logger::Logger>) ->
         SetTimeFlags::Default,
     );
 
-    fds.push(libc::pollfd {
```

Comment:
Keep on removing! Since ```tfd``` is no longer put in the ```fds``` array, it doesn't need to exist at all. So you can remove all the code that sets it up as well, and once you've removed that, you should find some other things that only exist because they are necessary for setting up ```tfd```, and then you can remove those as well.